src/react/test-utils/Root.tsx

```diff
@@ -0,0 +1,26 @@
+import type { PropsWithChildren } from 'react'
+import { MODUKBody } from '../MODUKBody/MODUKBody'
+
+type RootProps = {
+  exampleName?: string
+  component?: string
+}
+export const Root = ({ exampleName, component, children }: PropsWithChildren<RootProps>) => {
+  const titlePrefix = exampleName && component ? `${exampleName} – ${component} – ` : ''
+  return (
+    <html lang='en'>
+      <head>
+        <title>{`${titlePrefix}MOD.UK Frontend React`}</title>
+        <meta name='viewport' content='width=device-width, initial-scale=1, viewport-fit=cover' />
+        <meta httpEquiv='X-UA-Compatible' content='IE=edge' />
+        <link href='/css/index.css' rel='stylesheet' />
+      </head>
+      <MODUKBody>
+        <div id='root'>
+          {children}
+        </div>
+        <script src='/react-example-client/index.js'></script>
+      </MODUKBody>
+    </html>
+  )
+}
```

Comment:
I had to move the boilerplate markup from 11ty to react to test the body tag. 
I think we may have to put some documentation to hardcode `js-enabled` for devs who are use `CRA / create-vite`.